airbyte-server/src/main/java/io/airbyte/server/validators/DockerImageValidator.java

```diff
@@ -47,7 +47,7 @@ public void assertValidIntegrationImage(String dockerRepository, String imageTag
     try {
       schedulerHandler.getConnectorSpecification(imageName);
     } catch (Exception e) {
-      throw new KnownException(422, "Encountered an issue while validating input docker image: " + e.getMessage());
+      throw new KnownException(422, String.format("Encountered an issue while validating input docker image (%s): %s", imageName, e.getMessage()));
```

Comment:
help when debugging to sanity check that we are looking for the correct docker image.